examples/userVaultsWithOldErc721.ts

```diff
@@ -0,0 +1,82 @@
+import { Address, isAddress } from 'viem';
+
+import { approveNFT, setAllowances, testCollection, testTokenId, users } from './common';
+
+const USER_VAULT_CONTRACT_V6 = process.env.USER_VAULT_CONTRACT_V6 ?? '';
+const ANOTHER_COLLECTION = process.env.TEST_COLLECTION_2 as Address;
+const OLD_ERC721_COLLECTION = process.env.OLD_ERC721_COLLECTION as Address;
+const OLD_ERC721_TOKEN_ID = BigInt(Number(process.env.OLD_ERC721_TOKEN_ID));
+
+const userVaults = async () => {
+  const gondi = users[1];
+
+  if (
+    !isAddress(ANOTHER_COLLECTION) ||
+    !isAddress(USER_VAULT_CONTRACT_V6) ||
+    !isAddress(OLD_ERC721_COLLECTION)
+  ) {
+    throw new Error(`
+      TEST_COLLECTION_2 OR USER_VAULT_CONTRACT_V6 or OLD_ERC721_COLLECTION were not provided.
+      Please, provide addresses to run user vaults example.
+    `);
+  }
+  await approveNFT(gondi, USER_VAULT_CONTRACT_V6, testCollection.contractAddress);
+  await approveNFT(gondi, USER_VAULT_CONTRACT_V6, ANOTHER_COLLECTION);
+
+  console.log('Creating vault with nfts...');
+  const { vaultId, receipts } = await gondi.createUserVault({
+    userVaultAddress: USER_VAULT_CONTRACT_V6,
+    nfts: [
+      { collection: ANOTHER_COLLECTION, tokenIds: [testTokenId] },
+      { collection: testCollection.contractAddress, tokenIds: [testTokenId] },
+      { collection: OLD_ERC721_COLLECTION, tokenIds: [OLD_ERC721_TOKEN_ID], isOldErc721: true },
+    ],
+  });
```

Comment:
```suggestion
  const gondi = users[1];
  const vaultERC721 = [
      {collection: testCollection, tokenId: testTokenId},
      {collection: ANOTHER_COLLECTION, tokenId: testTokenId}
  ];
  const vaultOldERC721 = [
      {collection: ANOTHER_COLLECTION, tokenId: testTokenId}
      {collection: OLD_ERC721_COLLECTION, tokenId: OLD_ERC721_TOKEN_ID}
  ]
  cons vaultNfts = [
      ...vaultERC721,
      ...vaultOldERC721
  ]
  for ([name, address] of [
  ["TEST_COLLECTION_2", ANOTHER_COLLECTION],
  ["USER_VAULT_CONTRACT_V6", USER_VAULT_CONTRACT_V6],
  ["OLD_ERC721_COLLECTION", OLD_ERC721_COLLECTION],
  ])
  if (!isAddress(ANOTHER_COLLECTION)) {
    throw new Error(`
      ${name} were not provided.
      Please, provide addresses to run user vaults example.
    `);
  }
  for(const {collection} of vaultERC721) {
      await approveNFT(gondi, USER_VAULT_CONTRACT_V6, collection.contractAddress);
  }

  console.log('Creating vault with nfts...');
  const { vaultId, receipts } = await gondi.createUserVault({
    userVaultAddress: USER_VAULT_CONTRACT_V6,
    nfts: vaultNfts,
  });
```

Reply:
deleted and added contract var on prev example